contracts/mocks/CurveAPI.vy

```diff
@@ -0,0 +1,70 @@
+# @version 0.3.3
+"""
+@title api for coin
+@license MIT
+@author fiddy
+"""
+
+interface ERC20:
+    def decimals() -> uint256: view
+
+interface Pool:
+    def coins(id: uint256) -> address: view
+    def get_virtual_price() -> uint256: view
+
+
+MAX_COINS: constant(uint256) = 4
+
+
+@external
+def __init__():
+    pass
+
+
+@internal
+@view
+def _get_coin_decimal(coin: address) -> uint256:
```

Comment:
Best to rename `_get_coin_decimals` for consistency

Reply:
Removed this mock api contract.